reference/functions/pgroonga-highlight-html.md

```diff
@@ -300,9 +300,68 @@ SELECT
    from posts where title &@~ 'Search Words' or body &@~ 'Search Words' LIMIT 10 OFFSET 100;
 ```
 
+## Practical example 2: keyword search and highlight using with `NormalizerTable`
+
+If you specify `NormalizerTable`, like in this `CREATE INDEX USING pgroonga` document ([`How to use NormalizerTable` section][create-index-using-pgroonga]), the specified PGroonga index must have not only `TokenNgram` with `"report_source_location", true"` option but also both `Normalizer` and `NormalizerTable` with `"report_source_offset", true"` option for each.
+
+Here is an example (We use Japanese Name 'Saito' (`斉藤`) which has many variants):
+
+```sql
+CREATE TABLE memos (
+  content text
+);
+
+CREATE TABLE "synonyms" (
+   "target" text not null,
+   "normalized" text not null
+);
+
+INSERT INTO synonyms VALUES ('齊', '斉');
+INSERT INTO synonyms VALUES ('斎', '斉');
+INSERT INTO synonyms VALUES ('齋', '斉');
+
+INSERT INTO memos  (content)  VALUES ('斎藤さんの恋愛');
+INSERT INTO memos  (content)  VALUES ('斉藤さんの失恋');
+INSERT INTO memos  (content)  VALUES ('齋藤さんの片想い');
+
+CREATE INDEX pgroonga_synonyms_index ON synonyms
+ USING pgroonga (target pgroonga_text_term_search_ops_v2)
+                INCLUDE (normalized);
+
+CREATE INDEX pgroonga_memos_index
+          ON memos
+       USING pgroonga (content)
+        WITH (
+         tokenizer='TokenNgram(
+                          "unify_alphabet", false,
+                          "unify_symbol", false,
+                          "unify_digit", false,
+                          "report_source_location", true
+                        )',
```

Comment:
```suggestion
                      "unify_alphabet", false,
                      "unify_symbol", false,
                      "unify_digit", false,
                      "report_source_location", true
                    )',
```

Reply:
We need auto prettier for SQL👀

Reply:
Yes but it may be difficult because this is embed in Markdown document...